### PR TITLE
Update Gateway 2022.2 RC docs

### PR DESCRIPTION
**Pull request by spikecurtis**
With #3125 closed, this updates the docs to reference the release candidate version we're now seeing work.

### CODE REVIEW

File: docs/ides.md [hunk 1]
```diff
@@ -88,11 +88,11 @@ After installing Gateway on your local system, you may connect to a Coder worksp
...
-| Version   | Status      | Notes                                                      |
-|-----------|-------------|------------------------------------------------------------|
-| 2021.3.2  | Working     |                                                            |
-| 2022.1.1  | Working     | Windows clients are unable to connect to Linux workspace   |
-| 2022.2 RC | Not working | [GitHub Issue](https://github.com/coder/coder/issues/3125) |
+| Version   | Status  | Notes                                                    |
+|-----------|---------|----------------------------------------------------------|
+| 2021.3.2  | Working |                                                          |
+| 2022.1.1  | Working | Windows clients are unable to connect to Linux workspace |
+| 2022.2 RC | Working | Version >= 222.3345.108                                  |
```
Comment (bpmct) [inline docs/ides.md:95]: Not sure if we should leave a note here since our docs are version-controlled, but only worked for me in main (Coder >=v0.8.2)

**Review by bpmct — APPROVED** [review]
🙌🏼

File: docs/ides.md (hunk 1 above)
Comment (spikecurtis) [reply docs/ides.md:95]: We should be deploying docs corresponding to releases.

**Review by kylecarbs — APPROVED** [review]

**Review by ammario — APPROVED** [review]
Appreciate the docs changes

**Pull request merged**